internal/remoteconfig/remoteconfig_test.go

```diff
@@ -48,10 +49,11 @@ func TestRCClient(t *testing.T) {
 		// Skip for now as this needs a valid Repository object
 		t.Skip()
```

Comment:
Can you unskip those now that you have the unverified client?

Reply:
I thought I did, nice catch :+1: